sphinx/source/docs/user_guide/examples/extensions_putting_together_ts.py

```diff
@@ -96,7 +96,7 @@ class Custom(HTMLBox):
 
     slider = Instance(Slider)
 
-    margin = Override(default=5)
+    margin = Override(default=(5, 5, 5, 5))
```

Comment:
This is due to the signature processing calling `class_default` which does not take into account `accepts`. We can:

* leave it this way (override defaults have to match property type exactly)
* skip signature processing for extension models altogether
* figure out how to make the old version work

Reply:
I suppose using a `lambda` here would also work with things as-is, but that seems obtuse for this situation

Reply:
This does not really seem worth putting much effort into, at least not in this PR, and "override defaults have to match property type exactly" seems like a defensible policy in any case. Anyone who feels strongly can open a separate issue.